ethereum/rpc/backend/backend.go

```diff
@@ -294,8 +294,32 @@ func (e *EVMBackend) GetTransactionLogs(txHash common.Hash) ([]*ethtypes.Log, er
 
 // PendingTransactions returns the transactions that are in the transaction pool
```

Comment:
this should maybe state that only `MsgEthereumTx` are considered

Reply:
I changed it to return `sdk.Tx`, the rpc side still filters `MsgEthereumTx` out, seems better this way.